src/main/flight/mixer.c

```diff
@@ -916,7 +950,6 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
             throttle = constrainf(throttle, -motorMixMin, 1.0f - motorMixMax);
         }
     }
-
```

Comment:
Unnecessary empty line removal?

Reply:
hey @stawiski you live in sydney.. where abouts? wana fly some time ?

Reply:
Good catch

Reply:
> hey @stawiski you live in sydney.. where abouts? wana fly some time ?

Are you on Betaflight Slack group?
I'd DM you :)

Reply:
@stawiski  yeah I'm @skonk